android/src/main/java/com/android/simplSdk/SimplSdkModule.java

```diff
@@ -26,61 +26,98 @@ public SimplSdkModule(ReactApplicationContext reactContext) {
         super(reactContext);
     }
 
-    @Override
-    public String getName() {
-        return "SimplSdk";
+    @ReactMethod
+    public void isUserApproved(final ReadableMap map, final Callback successCallback, final Callback errorCallback){
+        final HashMap<String, String> approvalMap = MapUtils.toHashMap(map);
+        final HashMap<String, String> paramsMap = MapUtils.stringToHashMap(String stringMap);
+
+        try {
+            Simpl.init(getReactApplicationContext(), approvalMap.get("merchantId"));
+            if (approvalMap.get("isSandbox") == "true")
```

Comment:
shouldn't this be `boolean` instead of `String`?

Reply:
approvalMap values are strings.